UITableView_ASS/UITableView_ASS/SceneDelegate.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SceneDelegate.swift
+//  UITableView_ASS
+//
+//  Created by Mohd Raza on 14/05/22.
```

Comment:
Please avoid using your name anywhere in the project.
This is applicable to client projects as well.